doc.txt

```diff
@@ -1,3 +1,20 @@
+/**
+ * @defgroup config Compile time configurations
+ * @brief    List of compile-time configurations
+ *
+ *           This group contains all compile-time configurations macros that are
```

Comment:
The indentation looks weird to me here (it seems as if it would belong to `@brief`. Typically we just align the detailed section with the `@tag`s AFAIK.